src/ray/protobuf/direct_actor.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package ray.rpc;
+
+import "src/ray/protobuf/common.proto";
+
+message ReturnObject {
+  // Data of the object.
+  bytes data = 1;
+  // Metaata of the object.
+  bytes metadata = 2;
+}
+
+message PushTaskRequest {
+  // The ID of the task to be pushed.
+  bytes task_id = 1;
+  // The task to be pushed.
+  TaskSpec task_spec = 2;
+}
+
+message PushTaskReply {
+  // The IDs for the returned objects.
+  repeated bytes return_object_ids = 1;
```

Comment:
put the id inside the return object message?

Reply:
will do. thanks.